src/client/customers.rs

```diff
@@ -580,7 +782,36 @@ impl Client {
                 .chain_one("ledger_entry"),
         );
         let req = req.json(entry);
-        let res = self.send_request(req).await?;
-        Ok(res)
+        self.send_request(req).await
+    }
+
+    /// Fetch a day-by-day snapshot of a customer's costs.
+    pub async fn get_customer_costs(
+        &self,
+        id: &str,
+        params: &CustomerCostParams<'_>,
+    ) -> Result<Vec<CustomerCostBucket>, Error> {
+        let req = self.build_request(Method::GET, CUSTOMERS_PATH.chain_one(id).chain_one("costs"));
```

Comment:
```suggestion
        let req = self.build_request(Method::GET, CUSTOMERS_PATH.chain([id, "costs"]);
```

Reply:
The pattern we're following in this crate is to `chain_one` multiple times in series.